src/orchard/info.clj

```diff
@@ -65,32 +65,29 @@
   {:added "0.5"}
   [{:keys [dialect ns sym computed-ns unqualified-sym] :as opts}]
   {:pre [(= dialect :clj)]}
-  (let [ns (or ns computed-ns)]
-    (if-not (and ns (find-ns ns))
-      ;; Lookups in files whose namespaces don't exist yet should still be able
-      ;; to resolve built-ins and fully-qualified syms
-      (recur (assoc opts :ns 'clojure.core))
-      (or
-       ;; it's a special (special-symbol?)
-       (m/special-sym-meta sym)
-       ;; it's a var
-       (some-> ns (m/resolve-var sym) (m/var-meta))
-       ;; it's a Java constructor/static member symbol
-       (some-> ns (java/resolve-symbol sym))
-       ;; it's an unqualified sym maybe referred
-       (some-> ns (m/resolve-var unqualified-sym) (m/var-meta))
```

Comment:
Removed these two lines, I don't think they make sense. If one is explicitly querying `:sym foo/+`, then trying to resolve `+` without foo is not accurate.

The comment says `maybe referred`.  But `(some-> ns (m/resolve-var sym) (m/var-meta))` already handles refers, that's part of what `resolve/var`/`ns-resolve` does